CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `ModelUsage` to `ModelCard` class.
 - Added a way to specify extra parameters to the predictor in an `allennlp predict` call.
 - Added a way to initialize a `Vocabulary` from transformers models.
+- Added `LogWriter` class. `TensorBoardWriter` and `ConsoleWriter` inherit from this. 
```

Comment:
I think these notes should make it clear that `ConsoleWriter` is an addition as well